xapian-applications/omega/query.cc

```diff
@@ -1547,7 +1563,6 @@ eval(const string &fmt, const vector<string> &param)
 			ket += '>';
 		    }
 		}
-
```

Comment:
Thanks for the information. There must be many aspects of git that are very useful that I haven't used proactively and this is definitely one of them. Will make use of it from now on.